src/main/java/nextstep/courses/service/SessionService.java

```diff
@@ -30,16 +28,14 @@ public SessionService(SessionRepository sessionRepository, SessionCoverImageRepo
 
     public void establish(Session session) {
         sessionRepository.save(session);
-        sessionCoverImageRepository.saveAll(session.getCoverImage());
+        sessionCoverImageRepository.saveAll(session.getCoverImages());
     }
 
     public void enroll(Long sessionId, NsUser user) {
-        Session findSession = sessionRepository.findById(sessionId)
+        Session session = sessionRepository.findById(sessionId)
                 .orElseThrow(() -> new SessionNotFoundException(sessionId));
-        SessionStudents findStudents = sessionStudentRepository.findAllBySessionId(sessionId);
-        Session session = SessionFactory.get(findSession, findStudents.get());
 
-        Payment payment = paymentService.payment(sessionId, user.getId(), session.getEnrollment().getFee());
+        Payment payment = paymentService.payment(sessionId, user.getId(), session.getEnrollment().getFee().get());
```

Comment:
(참고만 해주세요)
서비스 코드에서도 getter가 많아지면 가독성이 떨어지고 도메인의 변경이 서비스에도 영향을 줄 수 있는데요
이미 아시겠지만 디미터 법칙을 적용한다면 변경의 영향을 최소화할 수 있습니다
```suggestion
        Payment payment = paymentService.payment(sessionId, user.getId(), session.fee());
```